client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,62 @@
+import { ChangeEvent, useState } from 'react';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import DeleteIcon from '@material-ui/icons/Delete';
+import useStyles from './useStyles';
+import { User } from '../../interface/User';
+import Avatar from '@material-ui/core/Avatar';
+
+interface Props {
+  loggedInUser: User;
+}
+
+const ProfilePhoto = ({ loggedInUser }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  const [image, setImage] = useState<any>({});
+
+  const handleUpload = (event: ChangeEvent<HTMLInputElement>) => {
+    event.preventDefault();
+    console.log(event.target.files);
```

Comment:
Now that your PR is ready for review to be merged into main, I would clean up and remove unnecessary console logs